app/_locales/nl/messages.json

```diff
@@ -276,6 +251,9 @@
   "encryptNewDen": {
     "message": "Versleutel je nieuwe DEN"
   },
+  "endOfFlowMessage8": {
+    "message": "MetaMask cannot recover your seedphrase. Learn more."
```

Comment:
Interesting - I wonder how this got here.

This message isn't even the correct English message (the "Learn more" part was moved to `endOfFlowMessage9`)

Reply:
I've found this in other locales as well (e.g. `pt_PT`)

Reply:
The way that I remove english strings is to compare to the source locale. 
Source locale in this case doesn't match though. I prefer to fix that in a follow up that's not code-generation based though.

Reply:
Nevermind, I'm not sure why that one is different, I thought it was something already upstream since there were some english ones upstream already.  I'll look into it more.

Reply:
Hmm. It looks like `ethereum-remote-client` does have the correct `endOfFlowMessage8` message - it's just that the change wasn't propagated to the other locales. I think your code-generation approach would work just fine if this was fixed in the `ethereum-remote-client` locales first (i.e. if this change was applied to all locales: https://github.com/brave/ethereum-remote-client/pull/61/files)

I guess this is another problem with leaving the English messages in each locale - that it's not clear how they'd be kept in sync. This is a bit of a tangent - it's none of my business how you keep your locales in-sync, but - maybe you could add a flag to each of the message objects that was used as a fallback? e.g. ` "name": {  "message": "...",  "description": "...",  "placeholders": { ... }, "fallback": true  }` Then you could re-sync them whenever a change was made in `en`.

Reply:
OK this one is fixed now.  I verified and there are no other modified strings in that repo. We'll avoid modifying strings directly in the future.

Reply:
Great, thanks!

Just to be clear though - modifying strings directly wasn't what happened in this case exactly (the change in https://github.com/brave/ethereum-remote-client/pull/61/files was done upstream first, which is greatly appreciated). The problem was that the English fallback strings in the other locales weren't updated to reflect that change. You probably understood this already - I just wanted to make sure.

Reply:
👍 yep